client/control/sass/control.scss

```diff
@@ -382,3 +382,12 @@
       white-space: nowrap;
     }
 }
+
+/* Recipe List */
+#recipe-list {
```

Comment:
We generally don't use ids for selectors where we can avoid it. I think we can add a classname to the recipe component instead.